packages/CollabCodeEdit-Tests.package/CCEEditTest.class/instance/testRemovedMethodCanBeRemoved.st

```diff
@@ -0,0 +1,6 @@
+tests
+testRemovedMethodCanBeRemoved
+	self compileTestMethod.
+	self assert: (testClass canUnderstand: testMethod).
+	(CCEMethodRemovedEdit newWith: testMethod in: testClass) apply.
+	self deny: (testClass canUnderstand: testMethod)
```

Comment:
I would suggest to also add a test for a new method. So that all three szenarios are tested

Reply:
Do you search for `CCEEditTest>>testAddedMethodWillBeApplied`?

Reply:
Okay found it. within this method i would add a line before everythink else to deny that the class understands the method.

Reply:
I don't see why this is necessary. If this would be the case it should not alter the behaviour of the test.